internal/bminventory/inventory.go

```diff
@@ -4141,7 +4141,8 @@ func (b *bareMetalInventory) GetCredentialsInternal(ctx context.Context, params
 	}
 	var consoleURL string
 	if operatorscommon.HasOperator(cluster.Cluster.MonitoredOperators, operators.OperatorConsole.Name) {
-		if !b.clusterApi.IsOperatorAvailable(&cluster, operators.OperatorConsole.Name) {
+		// For the agent-installer, the console URL needs to be available prior to the finalizing stage
+		if b.installerInvoker != "agent-installer" && !b.clusterApi.IsOperatorAvailable(&cluster, operators.OperatorConsole.Name) {
```

Comment:
Will that work for the interactive case? I'm not sure that the cluster operators are even _yet_ available when the rendezvous node will reboot to join the temporary control plane

Reply:
It looks like the monitored list is built statically (as pointed out by @bfournie ) so definitely it will not be an issue.

/lgtm